src/data/resources.yaml

```diff
@@ -200,23 +215,31 @@ standards:
     grants:
       - Phenomics First
     monarch_role: Core
-    citation: https://doi.org/10.1093/database/baac035
+    citation: 
+      Nicolas Matentzoglu, James P Balhoff, Susan M Bello, Chris Bizon, Matthew Brush, Tiffany J Callahan, Christopher G Chute, William D Duncan, Chris T Evelo, Davera Gabriel, John Graybeal, Alasdair Gray, Benjamin M Gyori, Melissa Haendel, Henriette Harmse, Nomi L Harris, Ian Harrow, Harshad B Hegde, Amelia L Hoyt, Charles T Hoyt, Dazhi Jiao, Ernesto Jiménez-Ruiz, Simon Jupp, Hyeongsik Kim, Sebastian Koehler, Thomas Liener, Qinqin Long, James Malone, James A McLaughlin, Julie A McMurry, Sierra Moxon, Monica C Munoz-Torres, David Osumi-Sutherland, James A Overton, Bjoern Peters, Tim Putman, Núria Queralt-Rosinach, Kent Shefchek, Harold Solbrig, Anne Thessen, Tania Tudorache, Nicole Vasilevsky, Alex H Wagner, Christopher J Mungall. A Simple Standard for Sharing Ontological Mappings (SSSOM). Database 2022: baac035. https://doi.org/10.1093/database/baac035
     contact:
         name: Nicolas Matentzoglu
         email: nicolas.matentzoglu@gmail.com
         orcid: https://orcid.org/0000-0002-7356-1779
         github: "@matentzn"
     website_include: true
+    title: Simple Standard for Ontology Mappings
+    type: tool
```

Comment:
please do not add "type" slot at all anywhere, this is implicit in the groupings of the resources.